### ISSUE TITLE
Ziovdendian Bodypaint - Designs of the Lupine - RaceMenu Overlays - Special Edition

### ISSUE THREAD

**Issue by LevelTenLuck** [tiltedphoques/Mod-Compatibility#72]
# Mod information

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/25204

#### Dependencies
- #37 

# Compatibility

>Body and face paints can only be seen by the player.

# Additional Information

>Could indicate all mod-added body and face paints do not show up for other players

**Comment by Para0x** [comment]
Incorrect. It's any mod-added body and face paints that are ESL flagged.

**Comment by lBleyl** [comment]
anyway only faces overlays in warpaint category sincs, just remove the ESL Flag.

**Comment by LevelTenLuck** [comment]
Just checked in SSEEdit and the mod is not ESL flagged that I could tell.

**Comment by Para0x** [comment]
Okay so body overlays will not sync. Face overlays definitely do sync though, as long as they aren't ESLified. Glad we're starting to narrow this down.

**Comment by LevelTenLuck** [comment]
In our experience, only the vanilla face overlays sync properly, but we haven't tested all of them.

**Comment by lBleyl** [comment]
in my tests all custom face overlays (in warpaints section) have worked fine and synced.
